delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/PermissionsRoutes.scala

```diff
@@ -41,50 +43,61 @@ final class PermissionsRoutes(identities: Identities, permissions: Permissions,
 
   def routes: Route =
     baseUriPrefix(baseUri.prefix) {
-      extractSubject { implicit subject =>
+      extractCaller { implicit caller =>
```

Comment:
In order to avoid doing this everywhere, we could have in the Subject companion object something like:
```scala
implicit def subjectFromCaller(implicit caller: Caller): Subject = caller.subject
```